test/e2e/antreapolicy_test.go

```diff
@@ -1695,6 +1695,173 @@ func testRejectServiceTraffic(t *testing.T, data *TestData) {
 	time.Sleep(networkPolicyDelay)
 }
 
+// testRejectLoopTraffic tests that a double direction rejection won't cause an infinite rejection loop.
```

Comment:
s/a double direction rejection/a reject action in both traffic directions

Reply:
Done.